ui/lib/theme.ts

```diff
@@ -96,6 +97,26 @@ export const GlobalStyle = createGlobalStyle`
   .auth-modal-size {
     min-height: 475px
   }
+  body::-webkit-scrollbar-track, .fancy-scroll::-webkit-scrollbar-track {
```

Comment:
This is so good that I'm wondering if we even want to limit it to a class name. Why not override every ugly windows scrollbar? But that's a question for another day I think. Really nice work